prog/vm/github_runner.rb

```diff
@@ -247,6 +247,18 @@ def setup_info
     # Remove comments and empty lines before sending them to the machine
     vm.sshable.cmd(command.gsub(/^(\s*# .*)?\n/, ""))
 
+    if github_runner.installation.project.get_ff_all_servers_dnsmasq
```

Comment:
With this change we send every DNS query concurrently to 4 servers. Two might be already enough. 
```
Dec 11 10:32:59 vhxnr40hj3dp5kq5s49t28xth2 dnsmasq[1877858]: query[A] ab32a8b09b10c1392caed903.com from 10.193.172.228
Dec 11 10:32:59 vhxnr40hj3dp5kq5s49t28xth2 dnsmasq[1877858]: forwarded ab32a8b09b10c1392caed903.com to 2606:4700:4700::1111
Dec 11 10:32:59 vhxnr40hj3dp5kq5s49t28xth2 dnsmasq[1877858]: forwarded ab32a8b09b10c1392caed903.com to 2606:4700:4700::1001
Dec 11 10:32:59 vhxnr40hj3dp5kq5s49t28xth2 dnsmasq[1877858]: forwarded ab32a8b09b10c1392caed903.com to 2620:fe::fe
Dec 11 10:32:59 vhxnr40hj3dp5kq5s49t28xth2 dnsmasq[1877858]: forwarded ab32a8b09b10c1392caed903.com to 2620:fe::9
```

Reply:
instead of rewriting the config it would be cleaner to write two different kinds of configs in vm_setup from the get go. If this is temporary maybe it's ok.

Reply:
The good thing is that with this dnsmasq config change DNS resolution become much more stable and latency more predictable in a simple experiment I did.

Reply:
You're right but I made a decision to completely remove quad9 and use CF and Google if we're gonna use only 2 endpoints. At this point, I don't trust them much.